pkg/database/realm.go

```diff
@@ -356,6 +357,30 @@ func NewRealmWithDefaults(name string) *Realm {
 	}
 }
 
+// DefaultSMSTextTemplate returns database.DefaultSMSTextTemplate.
+// Convenance for unitizing in HTML templates.
+func (r *Realm) DefaultSMSTextTemplate() string {
+	return DefaultSMSTextTemplate
+}
+
+// DefaultENXSMSTextTemplate returns database.DefaultENXSMSTextTemplate.
+// Convenance for unitizing in HTML templates.
+func (r *Realm) DefaultENXSMSTextTemplate() string {
+	return DefaultENXSMSTextTemplate
+}
+
```

Comment:
Maybe I'm missing something, but why not add a func on the realm like `DefaultSMSTextTemplate` that returns the right template based on whether ENX is enabled. That would simplify the javascript too, because you move the conditional into the realm.

Reply:
good point